.github/workflows/build_minimum_supported_swift_platforms.yml

```diff
@@ -0,0 +1,49 @@
+name: Build with Minimum Supported Xcode Versions
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+  actions: write
+
+jobs:
+  build-amplify-with-minimum-supported-xcode:
+    name: Build Amplify Swift for ${{ matrix.platform }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os-runner: macos-12
+            xcode-version: 14.0.1
+            platform: iOS
+
+          - os-runner: macos-12
+            xcode-version: 14.0.1
+            platform: macOS
+
+          - os-runner: macos-12
+            xcode-version: 14.0.1
+            platform: tvOS
+
+          - os-runner: macos-12
+            xcode-version: 14.0.1
+            platform: watchOS
+
+    uses: ./.github/workflows/build_amplify_swift.yml
```

Comment:
Calling this workflow here as is will result in the build cache being overwritten.

You could add a new optional boolean input that allows to skip the **Save the dependencies cache in main** step in `build_amplify_swift.yml`.

Reply:
thanks! will update it.